packages/client3/components/design-system/input/checkbox.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import { Checkbox } from "./checkbox";
+
+export default {
+  component: Checkbox,
+  title: "Components/Input/Checkbox",
```

Comment:
```suggestion
  title: "FAD/Components/Input/Checkbox",
```

Reply:
Unless we have more stories of checkbox, we don't actually need to "../Input/***Checkbox***"

Reply:
Yeah right, integrated the changes